rgislackbot/tests/api_test.py

```diff
@@ -0,0 +1,39 @@
+import unittest
```

Comment:
looking at the available unit test suites I'd like to use pytest, with tox if running on bare metal.